src/screens/Notification.js

```diff
@@ -0,0 +1,326 @@
+          <PropertyValue type="string">
+            <Example>"Your session has expired."</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="message">
+          <Description>
+            A custom message displayed under the title. Provides more
+            information about the Notification.
+          </Description>
+          <PropertyValue type="string">
+            <Example>"You were signed out due to inactivity."</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="status">
+          <Description>
+            An indicator that relays the current status of a Notification.
+          </Description>
+          <PropertyValue type="string">
+            <Example defaultValue>"unknown"</Example>
+            <Example>"normal"</Example>
+            <Example>"warning"</Example>
+            <Example>"critical"</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="toast">
+          <Description>
+            A boolean flag that indicates whether or not the Notification is a
+            Toast Notification.
+          </Description>
+          <GenericBoolFalse />
+        </Property>
+
+        <Property name="onClose">
+          <Description>
+            Click handler. Not setting this property causes the Notification to
+            not have a close button.
+          </Description>
+          <PropertyValue type="function">
+            <Example>{`(...args: any[]) => any`}</Example>
+          </PropertyValue>
+        </Property>
+      </Properties>
+
+      <ThemeDoc>
+        <Property name="notification.toast.container">
+          <Description>
+            Any valid Box prop for the Toast Notification container.
+          </Description>
+          <PropertyValue type="object">
+            <Description>
+              'background.color' is used for the container color. 'width' is
+              used for the default width of a Toast Notification.
+            </Description>
+            <Example defaultValue>
+              {`
+{
+    elevation: 'medium',
+    round: 'xsmall',
+    width: 'medium',
+    pad: { horizontal: 'small', vertical: 'xsmall' },
+    background: {
+        color: 'background-front',
+    }
+}
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.iconContainer">
+          <Description>Any valid Box prop for the icon container.</Description>
+          <PropertyValue type="object">
+            <Example defaultValue>
+              {`
+{
+    pad: { right: 'small' },
+}
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.textContainer">
+          <Description>Any valid Box prop for the text container.</Description>
+          <PropertyValue type="object">
+            <Example defaultValue>
+              {`
+{
+    gap: 'medium'
+},
```

Comment:
Remove trailing comma here
```suggestion
{
    gap: 'medium'
}
```